### ISSUE TITLE
Interface Design

### ISSUE THREAD

**Issue by dristi-sigdel** [subin123khanal/Library#2]
Color combination in Interface Design is poor. It should be of proper color contrast. So that users can read the text easily.

![image](https://user-images.githubusercontent.com/81402784/114895496-4698c280-9e2f-11eb-8dd8-f42c5c09ac9d.png)

**Comment by hasty-bloke** [comment]
![174323304_1425747031138878_9163571891504681689_n](https://user-images.githubusercontent.com/81402735/115030250-9a1a1780-9ee6-11eb-9dd1-d44d36b4277f.png)
Thank you for the feedback. The issue has been resolved. 
-Susan Shrestha
